src/arkode/arkode_erkstep_io.c

```diff
@@ -234,6 +207,244 @@ int ERKStepGetNumRelaxSolveIters(void* arkode_mem, long int* iters)
   return arkRelaxGetNumRelaxSolveIters(arkode_mem, iters);
 }
 
+/*===============================================================
+  DEPRECATED ERKStep optional input/output functions
```

Comment:
I did not see similar wrappers in ARKStep for deprecated functions

Reply:
Those have been added back in -- I either deleted them accidentally or it occurred when merging changes from upstream.